docs/src/main/paradox/common/caching.md

```diff
@@ -76,7 +76,7 @@ For more advanced usage you can create an @unidoc[LfuCache$] with settings
 specialized for your use case:
 
 Java
-:  @@snip [CachingDirectivesExamplesTest.java]($root$/src/test/java/docs/http/javadsl/server/directives/CachingDirectivesExamplesTest.java) { #create-cache-imports #caching-directives-import #keyer-function #create-cache }
+:  @@snip [CachingDirectivesExamplesTest.java]($root$/src/test/java/docs/http/javadsl/server/directives/CachingDirectivesExamplesTest.java) { #create-cache-imports #caching-directives-import #time-unit-import #keyer-function #create-cache}
```

Comment:
nitpicking over here, but there is a missing space before the `}`

Reply:
Done